src/SFA.DAS.ApplyService.Web/Views/Roatp/ApplicationApprovedAlreadyActive.cshtml

```diff
@@ -39,7 +39,7 @@
            
             <p class="govuk-body">
                 If your organisation has any post-application (on-boarding) tasks to complete,
-                you’ll receive an email within 5 working days from the ‘Business Operations Provider Management’ team explaining how to do this.
+                you'll receive an email within 5 working days from the ‘Business Operations Provider Management’ team explaining how to do this.
```

Comment:
you missed the ones around Business Operations Provider Management

Reply:
sorted